src/app/register/components/backend-error/backend-error.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, Input } from '@angular/core'
+
+enum RegisterBackendErrors {
+  'orcid.frontend.verify.duplicate_email',
+  'additionalEmailCantBePrimaryEmail',
+  'duplicatedAdditionalEmail',
```

Comment:
@leomendoza123 the property 'orcid.frontend.verify.deactivated_email' is missing and when you try to register with an unclaimed error throws the next message 'unclaimed@test.com already exists in our system as an unclaimed record. Would you like to <a href='https://dev.orcid.org/resend-claim?email=unclaimed%40test.com'>resend the claim email</a>?'

Reply:
This is expected behavior, I am adding the following comment to explain better on the code, please let me know if this is clear enough 

```
    // When the error text is not listed on the RegisterBackendErrors enum
    // the error message will be displayed as it comes from the backend
    // This is because the backend might return code or a text ready for the UI 
```

Reply:
Ok but I think we need to resize the area of the error messages if it's too long as the example I put and the property 'orcid.frontend.verify.deactivated_email' is a known error and it should be in the enum